cmd/coldImport.go

```diff
@@ -39,6 +39,8 @@ var coldImportCmd = &cobra.Command{
 
 Geth must be synced over all of the desired blocks and must not be running in order to execute this command.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		SubCommand = cmd.CalledAs()
+		LogWithCommand = *log.WithField("SubCommand", SubCommand)
```

Comment:
Is `SubCommand` what cobra command calls these?  Not a big deal, just wondering why we're using `SubCommand` rather than `Command`.

Reply:
Yes, cobra lightly refers to them as `subCommands` and sometimes child commands. So the root command (or command) would be `vulcanizedb` and the subCommand `headerSync` as an example.